### DISCUSSION TITLE
Is NetworkX ready to use with NumPy 2.0?

### DISCUSSION THREAD

**Discussion by padix-key** [networkx/networkx#7529] [Q&A] [answered]
[NumPy 2.0](https://numpy.org/doc/stable/release/2.0.0-notes.html) has been released a few weeks ago. In NetworkX' `pyproject.toml` the dependency is `numpy>=1.23`, so NumPy 2 would be installed. However, since NumPy 2.0 introduced some breaking changes and the `numpy>=1.23` version pin already existed prior to its release, I would like to ask if NetworkX in conjunction with NumPy 2.0 is already tested to work.

**Comment by dschult** [comment, marked as answer]
Yes, NetworkX should work just fine with numpy 2.0.  We have been testing against their pre-release code for quite some time.
